### PR TITLE
Resolve merge conflicts introduced by RuboCop and fix failing specs

### PR DESCRIPTION
**Pull request by smlance**

### CODE REVIEW

**Comment by smlance** [comment]
Looks like this is just failing on the path specs that are currently failing in 2-0-dev, so this should be good to merge.

**Comment by segiddins** [comment]
@smlance this is a history _mess_. Please rebase before merging

**Comment by smlance** [comment]
@segiddins All branches I've opened for 2-0-dev have this history mess.

**Comment by segiddins** [comment]
@smlance yes, since I rebased 2-0-dev. You'll need to rebase / cherry pick your branches against 2-0-dev in turn

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,7 @@ before_script: travis_retry rake spec:travis:deps
 branches:
   only:
     - master
-    - /.+-dev$/
-    - /.+-stable$/
+    - 2-0-dev
```
Comment (segiddins) [inline .travis.yml]: this is wrong

File: bin/bundle [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+
+# Exit cleanly from an early interrupt
+Signal.trap("INT") { exit 1 }
```
Comment (segiddins) [inline bin/bundle]: this is wrong - should be `exe/bundler`

File: lib/bundler/deployment.rb [hunk 3]
```diff
@@ -0,0 +1,66 @@
+$stderr.puts "DEPRECATION: Bundler no longer integrates with " \
+  "Capistrano, but Capistrano provides its own integration with " \
+  "Bundler via the capistrano-bundler gem. Use it instead."
+
+module Bundler
```
Comment (segiddins) [inline lib/bundler/deployment.rb]: this should be deleted

File: lib/bundler/dsl.rb [hunk 4]
```diff
@@ -230,7 +224,8 @@ def add_git_sources
 
       # TODO 2.0 remove this deprecated git source
       git_source(:bitbucket) do |repo_name|
-        warn_deprecated_git_source(:bitbucket, "https://#{user_name}@bitbucket.org/#{user_name}/#{repo_name}.git")
+        # Double quotes here make some dsl_spec.rb specs fail
+        warn_deprecated_git_source(:bitbucket, 'https://#{user_name}@bitbucket.org/#{user_name}/#{repo_name}.git')
```
Comment (segiddins) [inline lib/bundler/dsl.rb]: wrong

File: man/bundle-install.ronn [hunk 5]
```diff
@@ -9,7 +9,6 @@ bundle-install(1) -- Install the dependencies specified in your Gemfile
                  [--jobs=NUMBER]
                  [--local]
                  [--deployment]
-                 [--force]
```
Comment (segiddins) [inline man/bundle-install.ronn]: wrong?

**Comment by smlance** [comment]
Should I open a new PR, or should I merge directly into 2-0-dev?

File: spec/support/helpers.rb [hunk 6]
```diff
@@ -339,5 +339,17 @@ def process_file(pathname)
...
       File.open(pathname, "w") { |file| file.puts(changed_lines.join) }
     end
+
+    def with_env_vars(env_hash, &block)
+      current_values = {}
+      env_hash.each do |k,v|
+        current_values[k] = v
+        ENV[k] = v
+      end
+      block.call if block_given?
```
Comment (artofhuman) [inline spec/support/helpers.rb]: How about use `yield` instead `block.call` ?
`yield` more faster

Comment (segiddins) [inline spec/support/helpers.rb]: I actually prefer using explicit blocks / procs / lambdas than using yield

**Pull request merged**